2-mandatory/7-lane-names.js

```diff
@@ -6,7 +6,15 @@
   HINT: string and array methods that could be helpful (indexOf, filter)
 */
 
-function getLanes() {}
+let laneNames = [];
+
+
+function getLanes(roadNames) {
+  if (roadNames.includes("Lane")) {
```

Comment:
As the hint said, indexOf would be useful to help check the partial match in the string.